Week2/homework/js-exercises/ex-5-whoWantsDrink.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+console.log('EXERCISE 5 - Who wants a drink?');
+
+/* EXERCISE-5 =>  */
+
+/* We declared our empty array, then with for loop 
+we pushed our drinkTypes. Finally with back tick we logged our sentence.  */
+
+let drinkTray = [];
+const drinkTypes = ['cola', 'lemonade', 'water'];
+
+for (let i = 0; i < 5; ++i) {
+  drinkTray.push(drinkTypes[i % 3]);
+}
+
+console.log('Hey guys, I brought a ' + drinkTray.join(', ') + '!');
```

Comment:
This logic seems just right but it feels very unstable. For example, what happens if you have to iterate over the loop not 5 but 7 times. Then, we would have `cola` in it thrice, isn't it? This is because you don't have a check whether `drinkTray` already contains two instances of the item. 🤔